docs/multichain.md

```diff
@@ -2,20 +2,190 @@
 
 ## Introduction
 
-TODO
+One of the most important features of Simperby is its multichain
+interoperability. Any DAO (Decentralized Autonomous Organization) built with
+Simperby can control various Dapps (Decentralized Applications) deployed on
+various existing chains, such as Ethereum. This could include a treasury for the
```

Comment:
why?

Reply:
I was thinking for people who are not familiar with blockchains, it would be better to say blockchain rather than chains. Maybe fixing it to "existing blockchains"? (if removing existing seems changing the meaning.)

Reply:
I changed it to `existing blockchains`